codespell_lib/data/dictionary.txt

```diff
@@ -8552,10 +8552,15 @@ universtiy->university
 univesities->universities
 univesity->university
 unknonw->unknown
-unknow->unknown
+unknonws->unknowns
+unknow->unknown, unknow,
+unknows->unknowns, unknows,
 unknwon->unknown
+unknwons->unknowns
 unkonwn->unknown
+unkonwns->unknowns
 unkown->unknown
+unkowns->unknown
```

Comment:
this is likely also meant to be `unknowns`

Reply:
indeed! apologies and thanks.